README.md

```diff
@@ -38,16 +38,18 @@ In short, to get a major feature added, one usually first gets the RFC merged in
 1. Submit a pull request. As a pull request the RFC will receive design feedback from the larger community, and the author should be prepared to revise it in response.
 1. Build consensus and integrate feedback. RFCs that have broad support are much more likely to make progress than those that don't receive any comments.
 1. Eventually, the team will decide whether the RFC is a candidate for inclusion.
-1. RFCs that are candidates for inclusion will enter a "final comment period" lasting 3 calendar days. The beginning of this period will be signaled with a comment and tag on the RFCs pull request.
+1. RFCs that are candidates for inclusion will enter a "final comment period" lasting one week. The beginning of this period will be signaled with a comment and tag on the RFCs pull request. Upon request of a Babel core team member, this period can be extended by one additional week.
```

Comment:
This 3 days period was copied from [React's RFC process](https://github.com/reactjs/rfcs/pull/21). However, React's community is more active than ours, so it makes sense for us to have a longer period.